packages/server/api/repository/personality.js

```diff
@@ -15,8 +16,47 @@ module.exports = class PersonalityRepository {
         return newPersonality.save();
     }
 
-    static getById(personalityId) {
-        return Personality.findById(personalityId).populate('speechs', '_id title');
+    static async getById(personalityId) {
+        const personality = await Personality.findById(personalityId)
+            .populate('claims', '_id title');
+        if (personality) {
+            const stats = await this.getReviewStats(personalityId);
+            return Object.assign(personality.toObject(), { stats });
+        }
+        return personality;
+    }
+
+    static async getReviewStatsByClaims(id) {
+        const personality =  await Personality.findById(id);
+        return Promise.all(
+            personality.claims.map(async(claimId) => {
+                const reviews = await ClaimReview.aggregate([
+                    { $match: { claim: claimId } },
+                    { $group: { _id: "$classification", count: { $sum: 1 } } },
+                ]);
+
+                return { claimId, reviews };
+            })
+        ).then((result) => {
+            return result;
+        });
+    }
+
+    static async getReviewStats(id) {
+        const personality =  await Personality.findById(id);
+        const reviews = await ClaimReview.aggregate([
+            { $match: { personality: personality._id } },
+            { $group: { _id: "$classification", count: { $sum: 1 } } },
+        ]);
+        const total = reviews.reduce((agg, review) => {
+            agg += review.countç;
```

Comment:
I think the letter **Ç** not exist when you use `count`.

Reply:
Good catch!